ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -36,6 +36,24 @@ CREATE TABLE configuration_entries (
 
 CREATE INDEX idx_configuration_submission ON configuration_entries (submission_id);
 
+---------------------------------------------------------------------------------------------------
+-- User management tables
+---------------------------------------------------------------------------------------------------
+CREATE TABLE participant_users (
+    internal_id         INTEGER         GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    user_id             VARCHAR         NOT NULL UNIQUE,
+    primary_party       VARCHAR,
+    created_at          BIGINT          NOT NULL
+);
+
+CREATE TABLE participant_user_rights (
+    user_internal_id    INTEGER         NOT NULL REFERENCES participant_users (internal_id) ON DELETE CASCADE,
+    user_right          INTEGER         NOT NULL,
+    for_party           VARCHAR,
+    granted_at          BIGINT          NOT NULL,
```

Comment:
```suggestion
    granted_at          BIGINT          NOT NULL    DEFAULT CURRENT_TIMESTAMP,
```

Reply:
need to validate the EPOCH extraction. Taken literally from https://stackoverflow.com/questions/53474340/equivalent-of-unix-timestamp-function-for-h2-db/53886749